uqcsbot/events.py

```diff
@@ -171,7 +161,7 @@ class Events(commands.Cog):
     """
     def __init__(self, bot: UQCSBot):
         self.bot = bot
-        self.bot.schedule_task(self.scheduled_message, trigger='cron', hour=9, day_of_week="mon", timezone='Australia/Brisbane')
+        self.bot.schedule_task(self.scheduled_message, trigger='cron', hour=22, minute=32, timezone='Australia/Brisbane')
```

Comment:
Why has this changed from Monday 9am?

Reply:
Me forgetting that I changed this to ensure it was working 🤦